src/styles/Checkbox.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+@import "./Colors";
+
+.clr-checkbox-wrapper {
+    input[type="checkbox"] {
+        &:checked {
+            + label {
+                &::before {
+                    background: $BLUE;
+                }
+            }
+        }
+        &:focus {
+            + label {
+                &::before {
+                    border: 2px $LIGHTER-BLUE;
+                }
+            }
+        }
+        + label {
+            &::before {
+                height: 16px;
+                width: 16px;
```

Comment:
is it possible to convert px to rem ?

Reply:
Putting this in rem here might change checkbox size as per default font and in DDS checkbox size is set to 16px only